article/sections/_9_conclusion.tex

```diff
@@ -0,0 +1,42 @@
+\section{Conclusion}
+\label{sec:conclusion}
+
+Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 
+To date, choice modelers have not routinized the use of causal inference in the field of travel demand modeling.
+\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.
+
+In this chapter, we built upong their framework.
+We highlighted the importance of using causal graphs and causal inference methods in transportation demand modeling efforts and in choice modeling more generally.
+
+We presented a numerical exercise aiming to show the importance of  the data generating process in the estimation 
+of treatment effects from external interventions. 
+We showed, using the same outcome model, that different assumptions about the data generating processes could 
+lead to bias in estimates of the effect of interest.
+
+Data generating processes can be illustrated effectively using DAGs as shown by \citet{pearl_1995_causal}.
+We have shown that DAGs help researchers and practitioners represent their assumptions about the data generating process. 
+
+We presented a process allowing researchers and practitioners to construct causal graphs based on their expert opinion of the problem at hand. 
+We also presented methods for testing the implications encoded in the practitioner's proposed causal graph. 
+These implications might be either observed, latent, or both.
+
+Beyond testing the implications and assumptions encoded in a one's causal graph, we describe why causal discovery is important in discerning relationships between covariates one's data.
+We briefly presented several causal discovery algorithms used by
+other researchers in different fields. 
+We highlighted the use of one of these causal discovery algorithms on a simple example. 
+We showed that causal discovery methods help us test independence within our data, 
+develop a clearer picture about the uncertainty within the variables shown in the causal graph, 
+and expand the level and amount of tests performed on our data when compared to the tests implicated 
+by the structure of the researcher's proposed causal graph.
+
+We presented examples of latent confounding within the transportation demand modeling field and highlight 
```

Comment:
Suggesting a transition between the earlier set of topics and latent confounding.
```suggestion
Moving from general guidance to dealing with specific complexities facing choice modelers, we presented examples of latent confounding within the transportation demand modeling field.
```